### PR TITLE
Feat/remote meeting option

### PR DESCRIPTION
**Pull request by bearpersson**
## Explain the changes you’ve made
- Added checkbox component
- Added remote meeting option in bookingform

## Explain why these changes are made

Possability for remote meeting

## How to test

Concrete example:

1. Checkout this branch
2. Swap the application to run storybook
3. Fire upp the simulator by running the command `yarn ios`
4. ...etc.

## Tested environments

- [] Your local environment.
- [] Staging environment.

## Screenshots

(Optional)

Screenshots are helpful for UI-related changes. It could be an before and after change as an example.
Even backend code can benefit from a screenshot of the net change.

---

## Checklist for reviewers

Here are a few points that are useful for reviewers to check.

The short list of the (maybe) most important points are:

- Does this PR work as expected?
- Does the PR properly only concern a single feature/bug/thing?
- Is the code clear and concise?
- Will the changes impact existing functionality?
- Are there any linting errors (ESLint) that should be resolved?
- Do the automatic GitHub checks pass?

However it is good if a reviewer can be as thorough as possible,
within reasonable limits of course.

### Functionality

- Does this PR work as expected?
- If this is a new feature, does it add value, or is it a sign of feature-creep?
- Will the changes impact existing functionality?
- Does the changes create inconsistencies somewhere else?
- Does the code work on different platforms if applicable?
- How does the code handle if unexpected values are passed to functions etc.?

### Readability, syntax, formatting, style

- Is the code clear and concise?
- Has styling rules (Prettier) been applied to included files?
- Are functions/variables/etc. aptly named?

### Design

- Is the code properly planned and designed?
- Is code organized/separated in a logical fashion?
- Does the code follow clean code practices to a reasonable degree?
- Is the code self-documenting enough?
- Is the code in sync with existing patterns/technologies?
- Does the code reuse existing functionality where applicable?
- Is the code designed with reusability in mind?
- Does the code have hard-coded values that can be refactored?

### Patterns, idioms, tests, best practices

- Are there any linting errors (ESLint) that should be resolved?
- Is the code properly typed (TypeScript)?
- Does the code utilize language features to their full potential?

### Debugging, testing, configuring

- Is appropriate logging used?
- Have tests been added/modified to accommodate the changes?
- Is the feature sufficiently configurable in case tweaks are required in the future?

### Performance, reliability, scalability

- Is the code well optimized?
- Does the code scale?
- Does the code handle failure scenarios?

### Security

- Is there any obvious potential for exploits and security threats?
- Does code check proper authorization where applicable?

### Etiquettes

- Are commits atomic (does only one thing per commit)?
- Does the PR properly only concern a single feature/bug/thing?
- Are commit messages well-written?
- Do the automatic GitHub checks pass?

### CODE REVIEW

File: source/js/components/Checkbox/Checkbox.test.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  expect(container.querySelector('input')!).not.toBeChecked();
+});
+it('calls the onChange prop when clicked', () => {
+  const handleChange = jest.fn();
+
+  const { getByText } = render(<Checkbox id="checkbox" label="MyLabel" checked={false} onChange={handleChange} />);
+  fireEvent.click(getByText(/MyLabel/i));
+
+  expect(handleChange).toHaveBeenCalledTimes(1);
+});
```
Comment (karlknowit) [inline source/js/components/Checkbox/Checkbox.test.tsx]: Missing newlines between each test

Comment (karlknowit) [inline source/js/components/Checkbox/Checkbox.test.tsx:38]: Missing newlines between each test

File: source/js/services/__tests__/BookingService.test.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    const formData = formDataMock();
+    formData.comment.name = undefined;
+
+    const requestData = buildBookingRequest(timeSlotMock, formData);
+
+    expect(requestData.body).toEqual(
+      '<body><p>Du har fått en bokning. Klicka på Acceptera för att bekräfta bokningen.</p><p>Förnamn: Test</p><p>Efternamn: Test</p><p>Email: user@test.se</p><p>Telefon: 123456</p><p>Jag vill ansluta digitalt: Nej</p></body>',
+    );
+  });
+});
```
Comment (karlknowit) [inline source/js/services/__tests__/BookingService.test.ts:70]: Missing newline between each test

Comment (karlknowit) [inline source/js/services/__tests__/BookingService.test.ts:70]: Missing newline between each test

**Review by karlknowit — CHANGES_REQUESTED** [review]

File: source/js/services/__tests__/BookingService.test.ts (hunk 2 above)
Comment (bearpersson) [reply source/js/services/__tests__/BookingService.test.ts]: Fixed

Comment (bearpersson) [reply source/js/services/__tests__/BookingService.test.ts]: Fixed

File: source/js/components/Checkbox/Checkbox.test.tsx (hunk 1 above)
Comment (bearpersson) [reply source/js/components/Checkbox/Checkbox.test.tsx]: Fixed

**Review by karlknowit — APPROVED** [review]

**Pull request merged**